Dnn.AdminExperience/ClientSide/Pages.Web/src/utils.js

```diff
@@ -190,7 +190,9 @@ function formatDate(dateValue, longformat) {
         return "-";
     }
 
-    return Moment(dateValue).locale(utilities.getCulture()).format(longformat === true ? "LLL" : "L");
+    const localizedFormat = require('dayjs/plugin/localizedFormat');
```

Comment:
Can we move this up near the imports

Reply:
@valadas sure thing - I just pushed a new commit.

Reply:
Thank you